### DISCUSSION TITLE
Website API role

### DISCUSSION THREAD

**Discussion by Abdallah-Awwad** [directus-labs/agency-os#89] [Q&A]
Could someone clarify the purpose of the Website API role? Currently, we provide the admin token in the `.env` file, and the Website API role's access controls are all set to none. 
I couldn't find any related code to that role. 

Or simply, should I replace the admin token with the website API token and set specific permissions for the frontend instead? 😅

**Comment by aldoprogra** [comment]
I think as described in the dashboard,it is the role for the permissions the frontend has, eg. access blocks or files. Even if all is set to none, you can press in the access control -> "system collections" you can see some permissions.
